app/controllers/merchant/dashboard_controller.rb

```diff
@@ -25,4 +25,8 @@ def index
     @pending_data = @pending_orders.zip(@pending_items.zip(@pending_item_orders))
   end
 
+  def items
+    @items = Item.where(merchant_id: current_user.merchant_id)
+  end
+
```

Comment:
Glad we could just automatically repurpose code from above!

Reply:
Yes! I wonder if this means we can refactor the items action into the index action. Maybe that's not how this kind of code works....